tests/unit/timing/strategies/test_agentic_replay.py

```
@@ -2992,3 +2992,36 @@ async def test_profiling_setup_logs_rootless_lane_count(caplog):
 
     msgs = [r.getMessage() for r in caplog.records]
     assert any("rootless" in m and "1" in m for m in msgs), msgs
+
+
+@pytest.mark.asyncio
+async def test_accelerated_warmup_context_overflow_short_circuit() -> None:
+    trajectory = Trajectory(conversation_id="trace_0", start_turn_index=1)
+    strategy, issuer, scheduler, _ = _make_strategy(
```

Ruff (0.16.1)

[warning] Unpacked variable `scheduler` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)